.github/workflows/build-and-test-all.yml

```diff
@@ -120,6 +120,7 @@ jobs:
           path: ~/.ccache
           key: recursor-${{ matrix.sanitizers }}-ccache-${{ steps.get-stamp.outputs.stamp }}
           restore-keys: recursor-${{ matrix.sanitizers }}-ccache-
+      - run: sudo ../../builder-support/helpers/install_rust.sh
```

Comment:
I remember struggling a bit with that since I could not find out either the absolute path or the cwd to execute reach `install_rust.sh`. Will revisit.

Reply:
As long as our package build do not have access to `inv`. I see no way around having a shell script. We might consider installing rust into our build containers, though.

Reply:
For this workflow, we can add Rust to https://github.com/PowerDNS/base-pdns-ci-image, which saves time.

For package builds, the build containers are built fresh (from builder-support/dockerfiles/*) and we will always install Rust fresh. Making `inv` available there might make sense, but for now I'm fine with calling a shell script in those.